Financial_Data_Report/python_code/reporting_spark_v2.py

```diff
@@ -58,4 +59,8 @@
 #Stop the Spark session after having sent the metadata
 import time
 time.sleep(30)
-spark.stop()
\ No newline at end of file
+spark.stop()
+
+# Cleanup the metadata
```

Comment:
we're removing the data here i think

Reply:
I was following the comment from line 59 https://github.com/kensuio-oss/kensu-public-examples/pull/10/files/35fb2d186c4bd5011cc3d78e725e39859efd4593#diff-bec0d50bc07abb6d425d76a74c87a31fce83f5a5a1cca4591a28a5253999c5d7R59